test/acceptance/integration/steps_lib/general.py

```diff
@@ -95,6 +95,13 @@ def check_the_response_has_the_key_with_the_value(step, key, value):
 def check_the_response_has_the_key_with_the_value(step, key, value):
     assert check_key_value(world.responses[world.response_count].json(), key, value) == False
```

Comment:
== False is not necessary and is useful a comment in case of the assertion failed